src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,14 +1,21 @@
-import { Layout } from 'antd'
 import { motion } from 'framer-motion'
+import { Typography } from '@mui/material'
 import { animationProps } from '../../styles/animation'
-const { Content } = Layout
 
 export const Dashboard = () => {
   return (
-    <motion.div {...animationProps}>
-      <Layout>
-        <Content>Welcome!</Content>
-      </Layout>
+    <motion.div
```

Comment:
As mentioned in the MR before this one, we could maybe wrap this `motion.div` + `animationProps` into a separate component